frontend/craco.config.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
     },
   },
   jest: {
-    configure: jestConfig => {
+    configure: (jestConfig) => {
```

Comment:
Is it expected that the new version of prettier changes a lot of our formatting rules? I'm all for upgrading the dependency, but this PR has thousands of style changes, which I wouldn't expect. Can we preserve our current styling while still upgrading?

Reply:
Oh. I thought prettier didn't have any style configuration options, but after checking more carefully I found one option that solved most of the problems - `arrowParens`.

Reply:
But other than that, I think we'll avoid a lot of changes since the prettier is an opinionated code formatter.